grouping_server/social_auth/serializers.py

```diff
@@ -35,7 +37,15 @@ class VerifierSerializer(serializers.Serializer):
     def validate(self, attrs):
         cipher = AES.new(b'haha8787 I am not sure fjkfjkfjk',AES.MODE_ECB)
```

Comment:
Magic Number `haha8787 I am not sure fjkfjkfjk` in both frontEnd and BackEnd,  why not put in .env file.